log-viewer-frontend/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.logviewer</groupId>
+        <artifactId>log-parent</artifactId>
+        <version>0.1.7-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>log-viewer-frontend</artifactId>
+    
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.github.eirslett</groupId>
```

Comment:
Put `frontend-maven-plugin` under `build-frontend` maven profile, please. Sometimes I need to build the project without recompiling UI

Reply:
no problem! In this I think we could add more profiles with only the modules that you want to build. But we could discuss this in a different thread.

Reply:
Oh... you are right, I can exclude the whole log-viewer-frontend module using `mvn -pl !:log-viewer-frontend` instead of using the profile, so the profile is not neccessary.